### PR TITLE
Update Readme for better readability

### PR DESCRIPTION
**Pull request by zhangwei217245**
Update Readme for better readability

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,7 +241,13 @@ export LIBFABRIC_DIR="where/you/installed/your/libfabric"
 export LD_LIBRARY_PATH="$LIBFABRIC_DIR/lib:$MERCURY_DIR/lib:$LD_LIBRARY_PATH"
 export PATH="$PDC_DIR/include:$PDC_DIR/lib:LIBFABRIC_DIR/include:$LIBFABRIC_DIR/lib:$MERCURY_DIR/include:$MERCURY_DIR/lib:$PATH"
 ```
+
+-->
+
+## About Spack
+
 One can also manage the path with Spack, which is a lot more easier to load and unload these libraries.
```
Comment (sbyna) [inline README.md]: "lot more easier" --> "lot easier"

Comment (zhangwei217245) [reply README.md]: resolved. 

Also, the paragraphs after this line got revised.

File: README.md [hunk 2]
```diff
@@ -1,121 +1,229 @@
...
+git checkout tags/v1.11.2
+```
+
+Configure, compile and install:
+
+```bash
 ./autogen.sh
-./configure --prefix=$LIBFABRIC_DIR CC=gcc CFLAG="-O2"
-make -j8
+./configure --prefix=$LIBFABRIC_DIR --disable-efa --disable-sockets CC=cc CFLAG="-O2"
```
Comment (houjun) [inline README.md]: Can you add a note here that "--disable-efa --disable-sockets" are only needed on Cori/Perlmutter

File: README.md [hunk 3]
```diff
@@ -1,121 +1,229 @@
...
 ```
-# Install PDC
-One can replace mpicc to other available MPI compilers. -DCMAKE_C_FLAGS="-dynamic" is sometimes required for Cori. For example, on Cori, cc can be used to replace mpicc.
-ctest contains both sequential and MPI tests for the PDC settings. These can be used to perform regression tests.
+
+## Compile and Install PDC
+Now, it's time to compile and install PDC.
+
+* One can replace `mpicc` to other available MPI compilers. For example, on Cori, `cc` can be used to replace `mpicc`. 
+* `-DCMAKE_C_FLAGS="-dynamic"` used to be required for Cori, but now it is not necessary any more.
```
Comment (houjun) [inline README.md]: Can remove this line since it no longer applies.

**Pull request merged**